tests/phpunit/Unit/Components/ComponentTest.php

```diff
@@ -188,16 +183,14 @@ public function testGetDomElement() {
 	 * @covers ::getClassString
 	 */
 	public function testGetClassString_WithoutSetting() {
-
 		$chameleonTemplate = $this->getChameleonSkinTemplateStub();
 
-		$instance = $this->getMockForAbstractClass( $this->classUnderTest, array( $chameleonTemplate ) );
+		$instance = $this->getMockForAbstractClass( $this->classUnderTest, [ $chameleonTemplate ] );
 		$instance->expects( $this->any() )
 			->method( 'getHtml' )
 			->will( $this->returnValue( 'SomeHtml' ) );
 
-		$this->assertInternalType( 'string', $instance->getClassString() );
-
+		$this->assertIsString( $instance->getClassString() );
```

Comment:
As per failing CI, this breaks compat with MW 1.31 - 1.33 and possibly 1.34.

Reply:
I will switch this back. Did it only happen in the one place? I cannot see the logs from CI anymore.

Reply:
Huh - not sure what happened with the first log.

Second one shows fine: https://travis-ci.org/github/ProfessionalWiki/chameleon/jobs/683922292

3 places

Reply:
Can't use `assertIsString` anywhere

Reply:
Part of a set of breaking changes in PHPUnit that are quite annoying for extension devs wanting to retain compat with more than MW master.